### PR TITLE
Modify readme

### PR DESCRIPTION
**Pull request by agaDoD**
As a response to the candidates feedback, I decided to make a few minor changes to README files as a description improvement.

### CODE REVIEW

**Review by ksnietka — APPROVED** [review]
Much better! Currently, the README has a friendlier vibe compared to the old one, which was more passive (or even passive-aggressive xD).

File: tasks/task2/README.md [hunk 1]
```diff
@@ -1,25 +1,63 @@
-# Store
-Let's imagine that we are building the store app.
-It should be the simple store where we can just manage orders and products (as admins).
-Customers should be able to place the order.
+# Store App Task
+
+## Overview
+
+Welcome to the Store App task!
+In this exercise, you will be developing a simple store application where admins can manage products and orders, while customers can place orders.
```
Comment (mateuszdroidsonroids) [inline tasks/task2/README.md]: 🤔 i believe we do not have any tasks related to managing orders by admins, so maybe we can remove the `and orders` from this sentence?

**Review by mateuszdroidsonroids — APPROVED** [review]

File: tasks/task2/README.md [hunk 2]
```diff
@@ -1,25 +1,63 @@
...
+
+Once you have completed the task, ensure all tests pass and the functionality is working as expected. Submit your solution according to the instructions provided by your interviewer.
+
+### Tips for Success
+
+- Understand the Requirements: Make sure you understand the functional requirements before writing any code.
+- Test your code: Write tests to make sure it passes the requirements.
+- Code Quality: Follow best practices for clean, maintainable, and readable code.
+
+## Good luck, and happy coding!
```
Comment (SkuzmaN) [inline tasks/task2/README.md:54]: ❤️

File: tasks/task2/README.md [hunk 3]
```diff
@@ -1,25 +1,63 @@
...
+
+2. Add More Tests and Code:
+   Extend the current code in the `/store/index.ts` by adding more functionalities along with corresponding tests to make the store functional.
+
+3. Create Products Repository:
+   Implement a products repository that will handle storing and retrieving product information.
+
+4. Add User Roles:
+   Implement shopper and admin roles. Only admins should have the ability to add products. Customers should be able to place an order.
+
```
Comment (SkuzmaN) [inline tasks/task2/README.md]: I guess we can remove this task or suggest candidate how to solve this in the smart way

**Review by SkuzmaN — APPROVED** [review]

**Review by MIKE432 — APPROVED** [review]

**Pull request merged**